snakeviz/cli.py

```diff
@@ -103,14 +105,20 @@ def main(argv=sys.argv[1:]):
         print('No available port found.')
         return 1
 
+    url = "http://{0}:{1}/snakeviz/{2}".format(hostname, port, filename)
     print(('snakeviz web server started on %s:%d; enter Ctrl-C to exit' %
            (hostname, port)))
+    print(unquote_plus(url))
```

Comment:
The print is for convenience.  When passed to the browser directly it'll still be encoded.  I had no problem copy/pasting the URL into Chrome on Windows.